curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md

```diff
@@ -10,6 +10,7 @@ You can obtain values from an object, and set a value of a property within an ob
 These are classically called <dfn>getters</dfn> and <dfn>setters</dfn>.
 Getter functions are meant to simply return (get) the value of an object's private variable to the user without the user directly accessing the private variable.
 Setter functions are meant to modify (set) the value of an object's private variable based on the value passed into the setter function. This change could involve calculations, or even overwriting the previous value completely.
+<strong>Note</strong><br>It is convention to precede the name of a private variable with an underscore (<code>_</code>). The practice itself does not make a variable private.
```

Comment:
If you check in the view mode here, **Note** is in the same line as previous sentence, You can add a `<br>` after previous sentence.